lib/mosaico.rb

```diff
@@ -24,9 +24,9 @@ def initialize_with_retain(*args)
 module Mosaico
   DEFAULT_LOCALE = :en
 
-  autoload :LocalImageBackend,       'mosaico/local_image_backend'
```

Comment:
Can we keep the local backend autoloads? Otherwise anyone using the v1 functionality will have to require these files manually.

Reply:
Yeah, make sense. Didn't think about that.